pkg/kyverno/apply/command.go

```diff
@@ -93,6 +93,13 @@ To apply policy with variables:
 					values:
 					<variable1 in policy2>: <value>
 					<variable2 in policy2>: <value>
+		namespaceSelector:
+			- name: <namespace1 name>
+			labels:
+				<label key>: <label value>
+			- name: <namespace2 name>
+			labels:
+				<label key>: <label value>
```

Comment:
What is the scope of `namespaceSelector`? Does it apply to all policies?

Reply:
yes, `namespaceSelector` will be applied to all the policies. 

If a namespace `test` is mentions in multiple resources, on which the policy need to be applied, user need to pass the namespace details only once under namespaceSelector.